presto-parser/src/main/java/com/facebook/presto/sql/tree/LikePredicate.java

```diff
@@ -33,6 +33,11 @@ public LikePredicate(Expression value, Expression pattern, Expression escape)
         this(Optional.empty(), value, pattern, escape);
     }
 
+    public LikePredicate(Expression value, Expression pattern, Optional<Expression> escape)
```

Comment:
Please make addition of this constructor the first commit, before introducing changes in `ShowQueriesRewrite.java`

Reply:
Also, please change commit msg from `Add a constructor of LikePredicate` to sth like
```
Add LikePredicate constructor accepting escape as Optional
```

(Just saying "Add a constructor of LikePredicate" sounds like there was no constructor before 😉 )

Reply:
@findepi Thank you for your good advice. I will revert the code, change the code and create new pull request.